<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="MAVPUN1121205" failures="2" tests="2" name="Default test(failed)" time="0.038" errors="0" timestamp="2024-05-25T17:57:55 IST">
  <testcase classname="session40_RetryFailedTestCases.TestCasesExample" name="tc02" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at session40_RetryFailedTestCases.TestCasesExample.tc02(TestCasesExample.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- tc02 -->
  <testcase classname="session40_RetryFailedTestCases.TestCasesExample" name="tc01" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at session40_RetryFailedTestCases.TestCasesExample.tc01(TestCasesExample.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- tc01 -->
  <testcase classname="session40_RetryFailedTestCases.TestCasesExample" name="tc03" time="0.0">
    <ignored/>
  </testcase> <!-- tc03 -->
</testsuite> <!-- Default test(failed) -->
